tests/periph_gpio/main.c

```diff
@@ -272,6 +311,8 @@ static int bench(int argc, char **argv)
 
 static const shell_command_t shell_commands[] = {
     { "init_out", "init as output (push-pull mode)", init_out },
+    { "init_out_h", "init as high-output (push-pull mode)", init_out_h },
+    { "init_out_l", "init as low-output (push-pull mode)", init_out_l },
```

Comment:
Can you rephrase the "high-output" / "low-output"? To me, this sounds like a special output mode.

Reply:
Ok, reworded.